widgets/layout-slider/layout-slider.php

```diff
@@ -138,6 +138,12 @@ function get_widget_form(){
 						'label' => __( 'Responsive Height', 'so-widgets-bundle' ),
 					),
 
+					'vertically_align' => array(
+						'type' => 'checkbox',
+						'label' => __( 'Vertically align slide contents', 'so-widgets-bundle' ),
```

Comment:
Should we change the label? I'm not sure vertically align necessarily means vertical center. 

**Vertically center align slide contents**

That work for you?

Reply:
That works.

Reply:
Do you want this applied to the Hero widget also?

Reply:
Yes, please, apply to the Hero too.